performance/views/report.njk

```diff
@@ -8,7 +8,7 @@
 {% block pageTitle %}Performance test report - EDUCATION.GOV.UK{% endblock %}
 
 {% block head %}
-  <link href="https://design-system.service.gov.uk/stylesheets/main-8b722cf33a433600f925958f167685f3.css" rel="stylesheet">
+  <link href="https://design-system.service.gov.uk/stylesheets/main-302bcc7bda33eed8f5e6726a2d42cbde.css" rel="stylesheet">
```

Comment:
This will break at some point but as it's just for this page, not too big a concern.

Reply:
yeah, it does that regularly, want to look at packaging it into the output in the style and script tags, also want to add mermaid support for graphs, but will work on the actual reports once I have the breakpoint tests running